fluent/src/util.js

```diff
@@ -1,27 +1,22 @@
-function nonBlank(line) {
-  return !/^\s*$/.test(line);
-}
-
-function countIndent(line) {
-  const [indent] = line.match(/^\s*/);
-  return indent.length;
-}
+const RE_NOT_BLANK = /\S/;
+const RE_INDENTION = /^\s*/;
```

Comment:
I think both of these regexes should be clear that indention is just ' ', and not the rest of whitespace, `[ \f\n\r\t\v\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]` (says MDN)

Reply:
Hmm, I'm not sure how this should work exactly. The non-blank test is intended to filter out empty lines which would otherwise set the `commonLength` to `0`. Perhaps it would be OK to only support fully empty blank lines in this scenario? In other words, if a blank line has a few spaces in it, the `ftl` dedenter would not filter it out and take its indent into account when calculating the common indent?